grants/faq.md

```diff
@@ -0,0 +1,153 @@
+</details>
+
+<details>
+  <summary><b>A W3F member approved my application. Does that mean it is accepted?</b></summary>
+
+  Open Grants applications require one third of the committee to approve your pull request. Since we have many different members with different backgrounds and specializations, it is possible that the committee disagrees and your application gets rejected even though one or two members approved it.
+  The application is accepted once the pull request is merged.
+</details>
+
+<details>
+  <summary><b>A W3F member approved my application. Does that mean it is accepted?</b></summary>
+
+  Open Grants applications require one third of the committee to approve your pull request. Since we have many different members with different backgrounds and specializations, it is possible that committee members disagree on an issue and your application gets rejected even though one or two members approved it.
+  The application is accepted once the pull request is merged.
+</details>
+
+
+## 💰 General Grants
+
+<details>
+  <summary><b>How do I apply for a General Grant? I see a Google form, a Github repository, requests for a document…</b></summary>
+
+  _Every_ General Grant application requires a [form submission](https://docs.google.com/forms/d/e/1FAIpQLSfMfjiRmDQDRk-4OhNASM6BAKii7rz_B1jWtbCPkUh6N7M2ww/viewform). If you like the application to be completely private, you will have to provide all the details about your plans, team, milestones, financials, etc. in a document that you can attach to the form. Our [template](https://github.com/w3f/General-Grants-Program/blob/master/grants/grant_application_template.md) shows what kind of information we are looking for.
+  For (partially) public applications, you are free to submit this information as part of the pull request instead or in addition. What information goes where is up to you, as long as we get all the required information.
+</details>
+
+
+## 🚚 Milestone Delivery
+
+<details>
+  <summary><b>How do I submit a milestone?</b></summary>
+
+  For details, please refer to the milestone delivery guidelines for your grants program (Open Grants, General Grants). Generally speaking, the most important part of a delivery is a list of deliverables with links to their implementation/realisation (_ideally_ linking to a specific commit, so you can continue working on your repository without messing up your delivery and complicating our evaluation) and any additional notes you might have. The list of deliverables for each of your milestones should be defined in your grant agreement.
+</details>
+
+<details>
+  <summary><b>Can I submit two or more milestones at once?</b></summary>
+
+  You can. However, we strongly encourage you to submit your work in increments (milestones), so that you can be sure we didn’t misunderstand (an aspect of) your application, and you didn't make changes to your plan or delivery that would have required a reevaluation of the application.
```

Comment:
```suggestion
  You can. However, we strongly encourage you to submit your work in increments of one milestone each, so that you can be sure we didn’t misunderstand an aspect of your application and in order to minimise the impact of potential changes to your deliveries that we might request.
```